lib/api/v3/activities/activity_representer.rb

```diff
@@ -47,20 +48,20 @@ class ActivityRepresenter < ::API::Decorators::Single
 
         link :user do
           {
-            href: api_v3_paths.user(represented.user.id),
-            title: "#{represented.user.name} - #{represented.user.login}"
+            href: api_v3_paths.user(represented.user.id)
           }
         end
 
         link :update do
           {
-            href: api_v3_paths.activity(represented.id),
-            method: :patch,
-            title: "#{represented.id}"
```

Comment:
Because `16435` is not a title I would present to a user.

And I wont introduce a proper shiny formatting (including localization) for a title that no one (assumption) will ever display.